### DISCUSSION TITLE
How to tag devices

### DISCUSSION THREAD

**Discussion by thegreekman** [Security-Onion-Solutions/securityonion#7762] [Unsupported Versions]
Hello, how can I tag devices in security onion.
For example, I want kibana to tell me that an ip, such as, 192.168.1.1 is a firewall when it shows up in the logs

**Comment by thegreekman** [comment]
Im not sure if this is the proper way to do this, but I managed to add tags via modifying the 0009_inputs_beats.conf file.
```

filter
 {
        mutate {
          rename => {"@metadata" => "metadata"}
     
        }
       if [log][source][address] =~ "10.x.x.x*"{
         mutate{
                add_tag => "cisco"
        }
       }
      }
```
My issue was that the syslog traffic of the firewalls/switches was not reporting there hostnames on kibana so I had no idea what was what. Manually adding the above resolved my issue.

**Comment by thegreekman** [comment]
Bump. Curious to see if anyone knows of any other way to tag devices?

**Reply by dougburks** [reply]
Another option might be to add tags via Elasticsearch ingest:
https://docs.securityonion.net/en/2.3/elasticsearch.html#parsing
